server/plugin/command.go

```diff
@@ -47,13 +52,22 @@ type HandlerMap struct {
 	defaultHandler HandlerFunc
 }
 
+// command stores command specific information
+type command struct {
```

Comment:
`command` is a bit generic. Maybe `commandHandler´?

Reply:
It was called like that before, I could change it too.